tests/unit/lms/validation/authentication/_bearer_token_test.py

```diff
@@ -1,36 +1,38 @@
 import datetime
 
 import pytest
-from pyramid.testing import DummyRequest
 from webargs.pyramidparser import parser
 
+from lms.services.exceptions import ExpiredJWTError, InvalidJWTError
 from lms.validation import ValidationError
 from lms.validation.authentication import (
     BearerTokenSchema,
-    ExpiredJWTError,
     ExpiredSessionTokenError,
-    InvalidJWTError,
     InvalidSessionTokenError,
     MissingSessionTokenError,
 )
-from tests import factories
 
 
 class TestBearerTokenSchema:
-    def test_it_serializes_lti_users_into_bearer_tokens(self, lti_user, schema, _jwt):
+    def test_it_serializes_lti_users_into_bearer_tokens(
+        self, lti_user, schema, jwt_service
+    ):
```

Comment:
The use of `test_it` is a bit confusing on an object with multiple methods and ways of using it